src/calculateRentalCost.js

```diff
@@ -4,7 +4,24 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = 0;
```

Comment:
Consider using `const` instead of `let` for `totalCost` since it is not reassigned after being initialized.